internal/envoy/run_test.go

```diff
@@ -57,7 +57,7 @@ func TestRuntime_Run(t *testing.T) {
 			name: "func-e Ctrl+C",
 			args: []string{"-c", "envoy.yaml"},
 			// Don't warn the user when they exited the process
-			expectedStdout:   moreos.Sprintf("starting: %s -c envoy.yaml %s\nGET /ready HTTP/1.1\n", fakeEnvoy, adminFlag),
```

Comment:
we don't expect func-e to make HTTP requests, but if it accidentally did, this would show up!